java-client/src/main/java/org/opensearch/client/opensearch/cat/ClusterManagerRequest.java

```diff
@@ -43,28 +43,28 @@
 import java.util.HashMap;
 import java.util.Map;
 
-// typedef: cat.master.Request
+// typedef: cat.cluster_manager.Request
 
 /**
- * Returns information about the master node.
+ * Returns information about the cluster-manager node.
  *
  */
-
-public class MasterRequest extends CatRequestBase {
-	public MasterRequest() {
+public class ClusterManagerRequest extends CatRequestBase {
+	public ClusterManagerRequest() {
 	}
 
 	/**
-	 * Singleton instance for {@link MasterRequest}.
+	 * Singleton instance for {@link ClusterManagerRequest}.
 	 */
-	public static final MasterRequest _INSTANCE = new MasterRequest();
+	public static final ClusterManagerRequest _INSTANCE = new ClusterManagerRequest();
 
 	// ---------------------------------------------------------------------------------------------
 
 	/**
 	 * Endpoint "{@code cat.master}".
```

Comment:
If we have changed the API request in this PR
`master` -> `cluster_manager`

Reply:
Do you mean for the comment `cat.master`? `cat.master` is for the deprecated endpoint, `cat.cluster_manager` is for the endpoint.

Reply:
I see we are supporting the deprecated and the new endpoint to not break things out there. Makes sense. You can ignore my comment.